docs/users_guide/landice/test_groups/ismip6_forcing.rst

```diff
@@ -0,0 +1,109 @@
+.. _landice_ismip6_forcing:
+
+ismip6_forcing
+============
+
+The ``landice/ismip6_forcing`` test group implements processing of atmospheric
+and ocean forcing data of the Ice Sheet Model Intercomparison for CMIP6
+(ISMIP6) protocol. The ISMIP6 source data can be obtained by directly
+contacting the authors. Reference: https://www.climate-cryosphere.org/wiki/index.php?title=ISMIP6-Projections2300-Antarctica#A2.2_Retrieving_datasets_and_uploading_your_model_output
+
+The test group includes 3 test cases, ``atmosphere``, ``ocean_thermal`` and
+``ocean_basal``. All test cases are made up of a single main step,
+``ProcessSMB``,``ProcessThermalForcing`` and ``ProcessBasalMelt``,
+respectively. Each step has the local methods (functions) of remapping and
+renaming the original ismip6 data to the format that MALI can incorporate in
+its forward simulations. In remapping the data, all test cases import the
+method ``build_mapping_file`` to create or use scrip files
+of the source (ISMIP6) and destination (MALI) mesh depending on the existence
+of a mapping file. Approximated time for processing a single forcing file
+on Cori (single core) is 2, 1, and 5 minutes for the atmosphere, ocean basal,
+and ocean thermal testcase, respectively.
+
+config options
+--------------
+
+All three test cases share some set of default config options under the section
+``[ismip6_ais]`` and separate config options under the corresponding section:
```

Comment:
What is shown here are not the default config options, they are examples of modifying the config options like one might do in a user config file.  This example is important but it would be better to first show the default config options and then show a typical user config file.